sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceClientBuilder.java

```diff
@@ -298,6 +326,7 @@ public TableServiceClientBuilder sasToken(String sasToken) {
      *
      * @throws NullPointerException If {@code credential} is {@code null}.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores the value set in this method. We should probably add a note like you did in other methods on this same class.